src/system/SystemPacketBuffer.h

```diff
@@ -654,5 +658,27 @@ inline PacketBufferHandle PacketBuffer::Last()
     return PacketBufferHandle::Hold(p);
 }
 
+class EncryptedPacketBufferHandle : public PacketBufferHandle
+{
+public:
+    EncryptedPacketBufferHandle() : mMsgId(0), mPayloadLen(0) {}
+    EncryptedPacketBufferHandle(EncryptedPacketBufferHandle && aBuffer) : PacketBufferHandle(std::move(aBuffer)) {}
```

Comment:
Needs to initialize the other members from the incoming object.

Reply:
Acked